draft-ietf-tls-tls13.md

```diff
@@ -2011,10 +2011,10 @@ legacy_compression_method
 
 extensions
 : A list of extensions.  The ServerHello MUST only include extensions
-  which are required to establish the cryptographic context. Currently
-  the only such extensions are "key_share" and "pre_shared_key".
-  All current TLS 1.3 ServerHello messages will contain one of these
-  two extensions, or both when using a PSK with (EC)DHE key establishment.
+  which are required to establish the cryptographic context and negotiate
+  the protocol version. All TLS 1.3 ServerHello messages MUST contain the
+  "supported_versions" extension, and at least one of either the "pre_shared_key"
+  or "key_share" extensions, or both when using a PSK with (EC)DHE key establishment.
```

Comment:
We are going from "current" to "all MUST" w.r.t. having at least one of pre_shared_key and key_share, which seems to rule out future modes that use neither.  I'm not sure I want to commit us to that indefinitely.

Reply:
Updated.